c/tests/test_tables.c

```diff
@@ -403,6 +410,73 @@ test_table_collection_metadata(void)
     tsk_table_collection_free(&tc2);
 }
 
+static void
+test_table_collection_time_units(void)
+{
+    int ret;
+    tsk_table_collection_t tc1, tc2;
+
+    char example_time_units[100] = "An example of time units with unicode ⏰";
+    tsk_size_t example_time_units_length = (tsk_size_t) strlen(example_time_units);
```

Comment:
I'm not sure strlen works when you have unicode in general, it'll stop at the first embedded zero byte, right? Not sure putting the unicode in there does much anyway?

Reply:
It's fine in this test code, we don't do `strlen` anywhere with foreign string data though for that reason as you say.